resources/ajax_forms/getEbscoKbPackageImportForm.php

```diff
@@ -0,0 +1,358 @@
+<?php
+
+$packageId = filter_input(INPUT_GET, 'packageId', FILTER_SANITIZE_STRING);
+$vendorId = filter_input(INPUT_GET, 'vendorId', FILTER_SANITIZE_STRING);
+
+if(empty($packageId) || empty($vendorId)){
+    echo '<p>Missing Package or Vendor ID</p>';
+    exit;
+}
+
+$titleOptions = [
+    [
+        'text' => 'Selected titles only',
+        'value' => 'selected',
+        'description' => 'Import only the titles you have selected in EBSCO Kb. The package will be added as a parent resource.'
+    ],
+    [
+        'text' => 'No titles',
+        'value' => 'none',
+        'description' => 'Do not import any titles.'
+    ],
+    [
+        'text' => 'All titles',
+        'value' => 'all',
+        'description' => 'Import all the titles in the package. The package will be added as a parent resource.'
+    ]
+];
+
+$ebscoKb = EbscoKbService::getInstance();
+$package = $ebscoKb->getPackage($vendorId, $packageId);
+
+//get all acquisition types for output in drop down
+$acquisitionTypeArray = array();
+$acquisitionTypeObj = new AcquisitionType();
+$acquisitionTypeArray = $acquisitionTypeObj->sortedArray();
+
+//get all resource formats for output in drop down
+$resourceFormatArray = array();
+$resourceFormatObj = new ResourceFormat();
+$resourceFormatArray = $resourceFormatObj->sortedArray();
+
+//get all resource types for output in drop down
+$resourceTypeArray = array();
+$resourceTypeObj = new ResourceType();
+$resourceTypeArray = $resourceTypeObj->allAsArray();
+
+// organizations
+$config = new Configuration;
+$orgModule = $config->settings->organizationsModule == 'Y' ? true : false;
+if ($orgModule) {
+    // TODO: Once namespaces are implemented, these sql calls can be removed. Call the Orgzanization versions of these classes via their namespaces instead.
+    $dbService = new DBService;
+    $orgDbName = $config->settings->organizationsDatabaseName;
+    $orgQuery = "SELECT organizationID, `name`
+			FROM ".$orgDbName.".Organization
+			WHERE ebscoKbID = $package->vendorId
+			LIMIT 0,1";
+    $result = $dbService->processQuery($orgQuery, 'assoc');
+    $organization = isset($result['organizationID']) ? (object)['primaryKey' => $result['organizationID'], 'name' => $result['name']] : false;
+
+    $aliasTypeQuery = "SELECT aliasTypeID, shortName
+			FROM coral_organizations.AliasType";
```

Comment:
This shouldn't be a hard-coded database name. If users have customized their database name during the install process, this causes a fatal error during a package import.

Reply:
ugh, though i'd fixed that. thanks for the catch! It's updated now.